OWNERS

```diff
@@ -22,13 +22,21 @@ approvers:
   - miekg
   - yongtang
 
+product-security-team:
```

Comment:
this change can't be made to the OWNERS file just yet, because dreck doesn't support it

Reply:
I'll add some thing to dreck.

On second thought; does this need to be public?

Reply:
> this change can't be made to the OWNERS file just yet, because dreck doesn't support it
What is dreck ? You mean it needs to be ingested by a tool ?


> On second thought; does this need to be public?
You are right, I do not think it is mandatory to list the persons. But that would be more clear for everyone to know who is in this list and who is not.
NOTE: I just copied that behavior from other project.